src/main/java/org/opensearch/ad/rest/handler/ModelValidationActionHandler.java

```diff
@@ -675,15 +710,24 @@ private void windowDelayRecommendation(long latestTime) {
             listener
                 .onFailure(
                     new ADValidationException(
-                        CommonErrorMessages.WINDOW_DELAY_REC + minutesSinceLastStamp,
+                        String.format(Locale.ROOT, CommonErrorMessages.WINDOW_DELAY_REC, minutesSinceLastStamp, minutesSinceLastStamp),
                         DetectorValidationIssueType.WINDOW_DELAY,
                         ValidationAspect.MODEL,
                         new IntervalTimeConfiguration(minutesSinceLastStamp, ChronoUnit.MINUTES)
                     )
                 );
             return;
         }
-        listener.onResponse(null);
+        // This case has been reached if no interval or window delay recommendation was found and no configuration took down bucket
+        // rate bellow the config bucket rate minimum.
```

Comment:
Don't you mean no config was found to take the bucket rate **above** the minimum?

Reply:
Config validation has a lower minimum threshold (0.25) since we are looking to see if any configs drastically decreases bucket success rate below 25%. There are cases when with all configs applied that no interval rec was found to increase success rate above 75% but no single configurations when added reduced bucket rate below 25%. This means that the suggestion is too simply ingest more data if possible but we don't have anymore insight on the root cause. In the future this can be improved to also give interval recommendation that gave the best rate that was found anyways even if below .75 and we can also do more testing to change the .25 rate for configs checks and etc. I was planning on trying to make those changes only after this is merged. Probably only in time for 1.4 and not for 1.3

Reply:
Ok, that makes sense. Can you add more detail in the comment like you've mentioned here? I think mentioning the 2 different thresholds, and how neither was crossed, can help others understand when this scenario occurs.